src/views/v2/rsvp/ari/form/template/fields.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * This template renders the RSVP AR form fields.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/ari/form/template/fields.php
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @var Tribe__Tickets__Ticket_Object $rsvp The rsvp ticket object.
+ * @var int $post_id The post ID the RSVP is linked to.
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+$has_meta = get_post_meta( $rsvp->ID, '_tribe_tickets_meta_enabled', true );
+
+if ( empty( $has_meta ) ) {
+	return;
+}
+?>
+<div class="tribe-tickets__form">
+
+	<?php $this->template( 'v2/rsvp/ari/form/error', [ 'rsvp' => $rsvp ] ); ?>
+
+	<?php
+		/**
+		 * Allows injection of meta fields in the RSVP ARI form template.
+		 *
+		 * @since TBD
+		 *
+		 * @var int $post_id The post ID the RSVP is linked to.
+		 * @var Tribe__Tickets__Ticket_Object $rsvp The rsvp ticket object.
+		 */
+		do_action( 'tribe_tickets_rsvp_attendee_fields_template', $post_id, $rsvp );
```

Comment:
Is the primary difference between this view and `src/views/v2/rsvp/ari/form/fields.php` the action naming? It seems to me that they use an argument that's passed to the action to differentiate instead of the duplicated view.

Reply:
Yes, the action naming, serves to identify where the call it's coming from, and we can either set the attendee ID to 0 (first attendee form) or use the `{{data.attendee_id}}` when we call the field template: https://github.com/moderntribe/event-tickets-plus/pull/1011/files#diff-653ab140dd2fdeed0daae5941ff48aa7R108

that's why I also implemented `tribe_tickets_plus_meta_field_get_attendee_id()` with the default of `{{data.attendee_id}}`